gateway-server/src/main/java/org/apache/knox/gateway/services/CLIGatewayServices.java

```diff
@@ -70,16 +54,21 @@ public void init(GatewayConfig config, Map<String,String> options) throws Servic
     defaultAlias.setKeystoreService(ks);
     defaultAlias.init(config, options);
 
+    final RemoteConfigurationRegistryClientService registryClientService =
+        RemoteConfigurationRegistryClientServiceFactory.newInstance(config);
+    registryClientService.setAliasService(defaultAlias);
```

Comment:
Doesn't make sense for this to be set to the remote alias service since this could be remote itself. This uses the default alias service in case of digest authentication.

If digest auth and using remote alias service, then wouldn't be able to connect to ZK anyway.